### DISCUSSION TITLE
Clean install.

### DISCUSSION THREAD

**Discussion by Keltbear** [KlipperScreen/KlipperScreen#1389] [General]
So as the title says I did a clean image. and started over from scratch on a rebuild.  I went through the Klipperscreen install process and when prompted to install network manager I did below are the results:

Insalling NetworkManager for the network panel
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
network-manager is already the newest version (1.30.6-1+deb11u1).
0 upgraded, 0 newly installed, 0 to remove and 44 not upgraded.

Installer just stops here. Then a while later I get "software caused connection abort."
After rebooting Klipper screen is there. I can click on the networks, and it sees them. However, it doesn't connect to them.  I can click on the Wi-Fi and type in the password. hit save Then I get a prompt saying "insufficient permissions detected would you like Klipperscreen to try and solve the issue?" I hit accept and it just goes back to the prompt to enter the password.  

I can no longer connect to Wi-Fi but it can see it.

Update: I took my SD card out and put back "wpa_supplicant.conf"  this allowed my pi to get back on Wi-Fi. Now the network area on Klipperscreen says that network manager doesn't have access.  Tried the below. Didn't work.

in order to fix this polkit needs to be configured or disabled:
here is how to disable polkit for network-manager:

mkdir -p /etc/NetworkManager/conf.d
sudo nano /etc/NetworkManager/conf.d/any-user.conf
in the editor paste this:

[main]
auth-polkit=false
Then restart the service (or reboot):

systemctl restart NetworkManager.service
systemctl restart KlipperScreen.service

**Comment by Keltbear** [comment]
Any thoughts?

**Comment by alfrix** [comment]
after installing you may need to reconnect (if you were not using nm), i added the info to the docs and installer to prevent confusion

**Reply by Keltbear** [reply]
That was part of the issue. I could see the wifi and click on it. however it would not allow reconnection.  Even tried entering the password on the waveshare screen.
